slw_agent/fsm_specification.yaml

```diff
@@ -0,0 +1,31 @@
+alphabet_in:
```

Comment:
This should be a part of the skill package not the agent package

Reply:
not sure where it should be placed. used to generate initial template for the skill

doubt will it be useful in future. moved to skill folder


also accprding to scaffolding docs:
```
Now that the agent is in place, you can generate the basic structure of a [skill](https://open-aea.docs.autonolas.tech/skill/). For that, you need its [FSM](https://docs.autonolas.network/fsm/) specification. In this example, let's copy the contents of the [Hello World](https://docs.autonolas.network/hello_world_agent_service/) service FSM specification into a file called fsm_specification.yaml. * Put it inside your agent's directory *.
```